docs/crd_mysql_cluster.md

```diff
@@ -0,0 +1,61 @@
+| `kind`       | string                                    | Kind.                                             |
+| `metadata`   | [ObjectMeta]                              | Standard object's metadata.                       |
+| `spec`       | [MySQLClusterSpec](#MySQLClusterSpec)     | Specification of desired behavior of the cluster. |
+| `status`     | [MySQLClusterStatus](#MySQLClusterStatus) | Most recently observed status of the cluster.     |
+
+MySQLClusterSpec
+----------------
+
+| Field                         | Type                    | Required | Description                                                                                                                                                           |
+| ----------------------------- | ----------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `replicas`                    | int                     | No       | The number of instances. Available values are 1, 3, and 5. Default value is 1.                                                                                        |
+| `podTemplate`                 | [PodTemplateSpec]       | No       | `Pod` template for MySQL server container.                                                                                                                            |
+| `volumeClaimTemplate`         | [PersistentVolumeClaim] | No       | `PersistentVolumeClaim` template for MySQL server container.                                                                                                          |
+| `serviceTemplate`             | [ServiceSpec]           | No       | `Service` template for endpoints of MySQL server containers.                                                                                                          |
+| `mySQLConfigMapName`          | string                  | No       | `ConfigMap` name of MySQL config.                                                                                                                                     |
+| `rootPasswordSecretName`      | string                  | Yes      | `Secret` name for root user.                                                                                                                                          |
+| `replicationSourceSecretName` | string                  | Yes      | `Secret` name which contains replication source info. Keys must appear in [options].<br/> If this field is given, the `MySQLCluster` works as an intermediate master. |
+| `restore`                     | RestoreSpec             | No       | Spec to perform PiTR from existing cluster. If this field is filled, start restoring. This field is unable to be updated.                                             |
```

Comment:
Should be a pointer?  `*RestoreSpec`.

Reply:
Yes, the type should be a pointer when we implement it in Go.
In the document, however, it is enough to show `RestoreSpec` with the "required" column.